internal/mempool/mempool_test.go

```diff
@@ -2410,7 +2410,9 @@ func TestHandlesTSpends(t *testing.T) {
 		testPoolMembership(tc, tx, false, false)
 	}
 
-	expiry := standalone.CalculateTSpendExpiry(nextHeight, tvi, mul)
+	// We'll calculate an expiry for the tests such that voting starts at
```

Comment:
Would suggest just saying `Calculate an` to be more consistent with the comments in the vast majority of the rest of the code.